lib/ios/RNNNavigationButtons.m

```diff
@@ -44,6 +44,10 @@ -(void)setButtons:(NSArray*)buttons side:(NSString*)side animated:(BOOL)animated
 		if(barButtonItem) {
 			[barButtonItems addObject:barButtonItem];
 		}
+		UIColor* color = [self color:[RCTConvert UIColor:button[@"color"]] defaultColor:[defaultStyle.color getWithDefaultValue:nil]];
+		if (color) {
+			self.viewController.navigationController.navigationBar.tintColor = color;
```

Comment:
This handles the back button tint color. If anyone has a better way of handling this, please feel free to comment :D Feedback is much appreciated :)